snap/hooks/install

```diff
@@ -1,9 +1,28 @@
 #!/bin/bash -e
 
+# setup env variables - none of these paths are setup for hooks like they are for
+# apps in snapcraft, so we have to do this ourselves
```

Comment:
Are these env vars required to run jq in this hook?

Reply:
Yes

Reply:
done